storage/migrations/migrations/1_filters_table.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS filters (
+    FNAME VARCHAR(30) UNIQUE NOT NULL,
```

Comment:
Why you are using 'f' prefix, and upper case?

Reply:
I wanted to avoid using reserved words. now it's changed